k9mail/src/test/java/com/fsck/k9/crypto/OpenPgpApiHelperTest.java

```diff
@@ -0,0 +1,38 @@
+package com.fsck.k9.crypto;
+
+
+import com.fsck.k9.Identity;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE, sdk = 21)
+public class OpenPgpApiHelperTest {
+
+    @Test
+    public void buildUserId_withName_createsAccountName() {
```

Comment:
`buildUserId` already says what should happen. If you want to include it in the name of the test method anyway, please prefix with "should".